pkg/metric/metric.go

```diff
@@ -20,18 +22,19 @@ var (
 	}, []string{"path", "method"})
 )
 
+//we can use curry with for these two below to slice HttpDuration metric by label authen/author
 var (
 	AuthnFailed = promauto.NewCounterVec(prometheus.CounterOpts{
 		Name: "search_authn_failed_total",
 		Help: "The total number of authentication requests that has failed",
-	}, []string{"reason"})
+	}, []string{"status_code"})
 )
```

Comment:
Let's remove this metric, when authentication fails, the status code will be `403 Unauthorized`.
So we can get the same information using the `status_code` label.

Reply:
yes